lib/engine/g_1893/corporation.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require_relative '../corporation'
+
+module Engine
+  module G1893
+    class Corporation < Corporation
```

Comment:
delete this class

Reply:
Ok. I have moved the floatable attribute into the general corporation class.